public/app/player/player.js

```diff
@@ -21,6 +21,7 @@ angular.module('turntify.player', ['ngMaterial', 'ngDraggable', 'ngAnimate'])
   vm.name = "";
   vm.isPlaylistExist = true;    //this variable is for showing if no playlist exist
   vm.playlistCounter = 0;     //this variable is for showing no playlist selected or no song in that turnt level
+  vm.isPlaylistLoading = true; //this variable is for spinner when loading playlists at beginning
```

Comment:
give this variable a more descriptive name - even if it's something long, we have a lot of variables with 'playlist' in it.
This one could be misunderstood to mean the songs in the playlist are loading or something